### ISSUE TITLE
Ruby is not an appropriate mechanism for managing speech data

### ISSUE THREAD

**Issue by r12a** [w3c/i18n-activity#799]
**This is a tracker issue.** Only discuss things here if they are i18n WG internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ https://github.com/w3c/pronunciation/issues/47

**Comment by xfq** [comment]
I agree with @r12a. Although ruby annotation can be used to indicate pronunciation, it sometimes describes meaning rather than or in addition to pronunciation. See the following section of clreq for some examples: https://w3c.github.io/clreq/#h-indicating_meaning_or_other_information

For ruby annotation that is known to contain pronunciation info, it can be used indeed (with limitations).

**Comment by murata2makoto** [comment]
Some DAISY people in Japan strongly argue that ruby should not be used for T2S.

If ruby text is used instead of ruby base, the morphological analysis receives hiragana only and is likely to fail.  Such failures lead to unnatural accent.

**Comment by xfq** [comment]
Agreed. Although the morphological analysis issue does not exist in Chinese, there are other limitations as well, like the lack of SSML expressions.

**Comment by r12a** [comment]
Gentlemen, this thread is only intended to be meta-information about the progress of the issue at https://github.com/w3c/pronunciation/issues/47.  These sound like technical contributions that should be logged over there(?)
